examples/list-setters-simple/.expected/config.yaml

```diff
@@ -0,0 +1,4 @@
+exitCode: 0
+testType: eval
+image: gcr.io/kpt-fn/list-setters:unstable
+includeMetaResources: true
```

Comment:
Let's not include Kptfile in the example and make it simple with just resources.

Reply:
done